Modules/Core/Controllers/UserClientsController.php

```
@@ -2,96 +2,144 @@
-    public function delete($user_client_id)
+    public function delete(int $user_client_id): \Illuminate\Http\RedirectResponse
     {
-        $ref = (new UserClientsService())->getById($user_client_id);
-        (new UserClientsService())->delete($user_client_id);
-        redirect('user_clients/user/' . $ref->user_id);
+        $ref = $this->userClientService->find($user_client_id);
+        $this->userClientService->delete($user_client_id);
+        
+        return redirect()->route('user_clients.user', ['id' => $ref->user_id])
+            ->with('alert_success', trans('record_successfully_deleted'));
     }
```

PHPMD (2.15.0)

The parameter $user_client_id is not named in camelCase. (undefined)

(CamelCaseParameterName)